bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -419,6 +565,52 @@ spec:
           - get
           - patch
           - update
+        - apiGroups:
```

Comment:
not sure what is going on here, but these will be added during the bundle build and should not be here

Reply:
Removed.